Loop/Public/SwiftUI/LoopBinding.swift

```diff
@@ -0,0 +1,49 @@
+import SwiftUI
+import Combine
+import ReactiveSwift
+
+@available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+@propertyWrapper
+public struct LoopBinding<State, Event>: DynamicProperty {
+    @ObservedObject
+    private var subscription: SwiftUISubscription<State, Event>
+
+    private let loop: Loop<State, Event>
+
+    @inlinable
+    public var wrappedValue: State {
+        acknowledgedState
+    }
+
+    public var projectedValue: LoopBinding<State, Event> {
+        self
+    }
+
+    @usableFromInline
+    internal var acknowledgedState: State!
+
+    public init(_ loop: Loop<State, Event>) {
+        // The subscription can be copied without restrictions.
+        self.subscription = SwiftUISubscription()
+        self.loop = loop
+    }
+
+    public mutating func update() {
+        if subscription.hasStarted == false {
```

Comment:
this check can be moved into the `SwiftUISubscription` so we'll have one check e.g `startIfNeeded()`